views/admin/events.erb

```diff
@@ -32,7 +32,30 @@
         </ol>
          <section id="status-<%= tab %>" role="tabpanel" tabindex="-1" class="dcf-tabs-panel" aria-labelledby="tab-<%= tab %>">
             <div class="event-page">
-                <h4><%= tab.capitalize %> Event Items</h4>
+                    <h4><%= tab.capitalize %> Event Items</h4>
+                    <form class="dcf-form dcf-mb-6" id="find-controls" style="float: right;margin-top:-55px">   <%# Move the box up to be more closely aligned with "Event Items"%>
+                        <div class="dcf-grid dcf-col-gap-vw dcf-row-gap-6" style="float: right;">
+                            <div class="dcf-col-100% dcf-col-100%-end@md">
+                                <div style="float: right;">
+                                    <label for="event-type">Event Type:</label>
+                                    <div class="dropdown-container" style="display:flex;">
+                                        <select id="event-type" style="max-width:200px;min-width:50px;margin-top:10px;" tabindex="1" name="event_type" class="dcf-input-select">
+                                            <option value=""></option>
+                                            <% Event.type_options.each do |cat_id, name| %>
+                                                <option <%= 'selected="selected"' if cat_id == event_type.to_i %> value="<%= cat_id %>"><%= name %></option>
+                                            <% end %>
+                                        </select>
+
+                                        <%# Assign the selected type to the `event_type` variable. %>
+                                        <input type="hidden" name="hidden-event-type" id="hidden-event-type" value="<%= event_type %>">
```

Comment:
You don't need this hidden field. Hidden fields are typically just used to get values that are passed from the .rb into javascript. So like if you wanted to use that event_type variable value inside of some javascript code you would use a hidden field.

Reply:
Sponsors decided they do not want the Event Type dropdown so all of the related code has been removed. So this is resolved.